COPYRIGHT.txt

```diff
@@ -173,6 +173,26 @@ Comment: The FreeType Project
 Copyright: 1996-2017, David Turner, Robert Wilhelm, and Werner Lemberg.
 License: FTL
 
+Files: ./thirdparty/harfbuzz/
```

Comment:
Sections are ordered alphabetically based on the "Files" path, so this should go after `./thirdparty/glad/`.